src/evm/initialize-provider.ts

```diff
@@ -55,7 +54,15 @@ export const initializeProvider = async (
     const log = node.logger.pend('ERROR', message);
     return [[log], null];
   }
-  const gasTargetMessage = `Gas target for chainId ${chain.id}: ${JSON.stringify(gasTarget)}`;
+  let gasTargetMessage;
```

Comment:
There's an opportunity to make this a separate function - but I have a feeling this will be replaced with an Airnode dependency, so probably unimportant.

Reply:
I basically copy pasted that code snipped from [airnode-node](https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/evm/handlers/process-transactions.ts#L45-L55) but you make a good point 🤔 

Since getGasPrice() is already returning an array of logs (return type is LogsData) then maybe it would be a good idea to include that snipped in the logs returned by this function instead of doing that formatting on the caller side. WDYT?